BackBLAST.py

```diff
@@ -90,119 +78,135 @@ def createProteomeHash(ProteomeFile):
     return ProteomeHash
 
 
-# ===========================================================================================================
-# Main program code:
-# House keeping...
-argsCheck(4)  # Checks if the number of arguments are correct.
+def main(args):
+    # ===========================================================================================================
+    # Main program code:
+    # House keeping...
+
+    queryFile = args.gene_cluster
+    queryBLASTDBFile = args.query_proteome
+    subjectBLASTDBFile = args.subject_proteome
```

Comment:
Note that these are no longer BLASTDBFiles but are .faa files, correct? Variable names should get refactored at some point for clarity.

Reply:
I also want to move everything under_score_case to comply with pep8. https://www.python.org/dev/peps/pep-0008/